compareECLFiles.cmake

```diff
@@ -99,7 +99,9 @@ set(abs_tol 2e-2)
 set(rel_tol 1e-5)
 
 add_test_compareECLFiles(spe1 SPE1CASE2 flow_ebos ${abs_tol} ${rel_tol} compareECLFiles "")
+add_test_compareECLFiles(spe1_2p SPE1CASE2_2P flow_ebos ${abs_tol} ${rel_tol} compareECLFiles "" spe1)
 add_test_compareECLFiles(spe1 SPE1CASE2 flow_legacy ${abs_tol} ${rel_tol} compareECLFiles "")
+add_test_compareECLFiles(spe1_2p SPE1CASE2_2P flow_legacy ${abs_tol} ${rel_tol} compareECLFiles "" spe1)
```

Comment:
this adds the actual tests using the macro. it should be pretty self-explanatory;
param1 = a case name, also serves as opm-data subdirectory name unless otherwise specified.
param2 = input file name from opm-data directory
param3 = simulator executable
param4,5 = tolerances to use in comparisons
param6,7 = some decoration due to code reuse
param8 = optionally the name of the directory in opm-data if casename does not match (see first commit in pr).